.eslintrc.json

```diff
@@ -8,7 +8,7 @@
     "array-callback-return": 1,
     "arrow-parens": [
       1,
-      "as-needed"
+      "always"
```

Comment:
So, yeah.

You might argue, but I am strongly of the opinion that we should always have parens for arrow functions. It doesn't add many characters, and can solve possible issues in the future (when adding a new parameter, you could forget about the parens etc).

I just think it's much better to do it.